configs/app.dev.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@michelled thanks!
I guess that we should include into the README file the instructions to run the app with a specific config.